src/main/groovy/net/minecrell/vanillagradle/BaseVanillaPlugin.groovy

```diff
@@ -46,6 +48,18 @@ abstract class BaseVanillaPlugin<T extends UserExtension> extends UserBasePlugin
         project.with {
             ProcessJarTask binDeobf = tasks.deobfBinJar
             ProcessJarTask decompDeobf = tasks.deobfuscateJar
+            DecompileTask decompile = tasks.decompile
+
+            // bin jar
+            def binName = getBinDepName() + "_${project.name}-{MC_VERSION}.jar";
```

Comment:
Yes, but the name is wrong. We want to include the project name if its dirty, so we reset it.